apps/dev-portal-web/domains/docs/components/widgets/AmountDistributionCalculator/DistributionItem.tsx

```diff
@@ -0,0 +1,110 @@
+import { Col, Row, RowProps } from 'antd'
+import React, { useEffect, useState } from 'react'
+import { useIntl } from 'react-intl'
+
+import type { TDistributionItem } from '@open-condo/billing/utils/paymentSplitter'
+import { Input, Switch, Typography } from '@open-condo/ui'
+
+import styles from './distributionItem.module.css'
+
+const GUTTER_COMPACT_ROW: RowProps['gutter'] = [8, 8]
+
+type DistributionItemProps = {
+    index: number
+    item: TDistributionItem
+    onUpdate: (index: number, item: TDistributionItem) => void
+}
+
+export const DistributionItem: React.FC<DistributionItemProps> = (props) => {
+    const { index, item, onUpdate } = props
+
+    const intl = useIntl()
+
+    const TinTitle = intl.formatMessage({ id: 'docs.widgets.amountDistributionCalculator.tin' })
```

Comment:
```suggestion
    const TinTitle = intl.formatMessage({ id: 'docs.widgets.amountDistributionCalculator.distributionItem.fields.tin.label' })
```

Reply:
No. I do not need to link translate strings to the DOM structure. I just need the word "TIN".

Reply:
That's how translations are organized in new apps.

Reply:
It means if I use some translation string in different components I'll duplicate the same translation into another translation key.